site/component/manifests/wordpress.pp

```diff
@@ -0,0 +1,37 @@
+class component::wordpress (
+  $path             = hiera('path', '/var/www/app_name'),
+  $vhost            = hiera('vhost', 'app-name.dev'),
+  $vhost_port       = 80,
+  $ssl              = false,
+  $mutisite         = false
+) {
+  if $ssl {
+    exec {'create_self_signed_sslcert':
+      command => "openssl req -newkey rsa:2048 -nodes -keyout ${vhost}.key  -x509 -days 365 -out ${vhost}.crt -subj '/CN=${vhost}'",
+      creates => [ "/etc/ssl/${vhost}.key", "/etc/${vhost}.crt", ],
+      path    => ["/usr/bin", "/usr/sbin"],
+      # require => Resource["${vhost}-${vhost_port}-wordpress"]
```

Comment:
Please cleanup this comment as it is non functional code and does not explain anything